.github/workflows/ci.yml

```diff
@@ -280,3 +280,27 @@ jobs:
         with:
           sarif_file: results.sarif
           category: zizmor
+
+  result:
```

Comment:
Good idea!

Reply:
I stole it from the internet when trying to figure out how to have different requirements for queueing and the final merge :) But moving more things out of the repository settings seems good for all of us as well.